scripts/setup.sh

```diff
@@ -6,17 +6,28 @@
 #
 # Commands that may fail have "|| return" to avoid continuing or interfering with terminal.
 
-if [ ! -z $NVM_DIR ]; then # skip nvm steps if not available
-  echo 'Configuring node 12...'
-  source ~/.nvm/nvm.sh
-  nvm install || return
+# Install yarn if it does not exist.
+if ! which yarn > /dev/null; then
+  echo 'yarn is required for setup, installing...'
+  if ! which brew > /dev/null; then
+    echo 'brew is required to install yarn, see https://docs.brew.sh/Installation'
+    exit 0
+  fi
+  brew install yarn
+fi
+
+if ! which memcached > /dev/null; then
+  brew install memcached
 fi
 
-echo 'Installing yarn and memcached...'
-brew bundle || return
+if [[ ! -z $NVM_DIR ]]; then # skip if nvm is not available
+  echo "Installing Node..."
+  source ~/.nvm/nvm.sh
+  nvm install
+fi
 
-echo 'Installing node packages...'
-yarn install || return
+echo "Installing dependencies..."
+yarn install || echo 'Unable to install dependencies using yarn!'
```

Comment:
Let's let this fail organically instead of echo and continue. If yarn does not install deps, the project wouldn't run anyway...
```suggestion
yarn install
```